app/models/storage_resource.rb

```diff
@@ -0,0 +1,24 @@
+class StorageResource < ApplicationRecord
+  include NewWithTypeStiMixin
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
```

Comment:
We don't need the AsyncDeleteMixin for something which is created/deleted by ems_refresh